app/models/document.rb

```diff
@@ -31,43 +31,27 @@ def self.get_all( current_user )
     if @leaf_manifest
       Document.import_manifest(@leaf_manifest, root_node)
     end
-   
+
   end
 
-  def tei_xml
-    xml_string = ""
-    tei_xml = ""
-    self.document_sections.each do |section|
-      if section.subsections.present?
-        section.subsections.each do |subsec|
-          if subsec.document_node.present?
-            if subsec.document_node.child_nodes.present?
-              subsec.document_node.child_nodes.order(:position).each do |child|
-                transcription = Transcription.find_by(leaf_id: child.leaf_id)
-                if transcription.present?
-                  if transcription.content.present?
-                    xml_string = "#{xml_string}#{transcription.content}"
-                  end
-                end
-              end
-            end
-          end
-        end
-      elsif section.document_node.present?
-        if section.document_node.child_nodes.present?
-          section.document_node.child_nodes.order(:position).each do |child|
-            transcription = Transcription.find_by(leaf_id: child.leaf_id)
-            if transcription.present?
-              if transcription.content.present?
-                xml_string = "#{xml_string}#{transcription.content}"
-              end
-            end
-          end
-        end
-      end
+  def append_node(node, xml)
```

Comment:
This is the only real change, everything else is whitespace. Using the "Hide whitespace" option is helpful.